statsmodels/genmod/cov_struct.py

```diff
@@ -938,6 +951,14 @@ class Equivalence(CovStruct):
     using this covariance structure, missing data should be removed
     before constructing the model.
 
+    If using `labels`, after a model is defined using the covariance
+    structure it is possible to remove a label pair from the second
+    level of the `pairs` dictionary to force the corresponding
+    covariance to be zero.
+
+    The `pairs` dictionary is modified by the function so the user
+    should pass a copy if it is needed elsewhere.
```

Comment:
I'll make a copy so this isn't needed.

On Wed, Jan 21, 2015 at 9:24 PM, Josef Perktold <notifications@github.com>
wrote:

> In statsmodels/genmod/cov_struct.py
> <https://github.com/statsmodels/statsmodels/pull/2188#discussion_r23351743>
> :
>
> > @@ -938,6 +951,14 @@ class Equivalence(CovStruct):
> >      using this covariance structure, missing data should be removed
> >      before constructing the model.
> >
> > +    If using `labels`, after a model is defined using the covariance
> > +    structure it is possible to remove a label pair from the second
> > +    level of the `pairs` dictionary to force the corresponding
> > +    covariance to be zero.
> > +
> > +    The `pairs` dictionary is modified by the function so the user
> > +    should pass a copy if it is needed elsewhere.
>
> I need to think about this, in general we avoid changing original arrays
> in functions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/statsmodels/statsmodels/pull/2188/files#r23351743>.
>